src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -12,25 +12,34 @@
   private CANSparkMax intakeMotor;
   private DoubleSolenoid intakePiston;
 
+  private boolean deployed;
+
   public Intake()
   {
     intakeMotor = new CANSparkMax(Constants.INTAKE_CAN_SPARK, MotorType.kBrushless); // Motor is Brushless
     intakePiston = new DoubleSolenoid(Constants.INTAKE_PISTON_DEPLOY, Constants.INTAKE_PISTON_RETRACT);
+
```

Comment:
Why the random space here?  This is a relatively short function so you don't need extra spaces.  It doesn't make it any more readable.